src/Microsoft.SqlTools.ManagedBatchParser/ReliableConnection/SqlConnectionHelperScripts.cs

```diff
@@ -56,7 +56,7 @@ @rc 			int
 END TRY
 BEGIN CATCH
 DECLARE @endpoint VARCHAR(max)
-select @endpoint = CONVERT(VARCHAR(max),SERVERPROPERTY('ControllerEndpoint'))
+select @endpoint = ISNULL(CONVERT(VARCHAR(max),SERVERPROPERTY('ControllerEndpoint')), '')
```

Comment:
Did you check whether this is changing whether we consider a server a BDC or not? I think this change might be breaking that so that it thinks servers are BDC's when they actually aren't. 

Easy way to tell is for your test server do you see an HDFS node under OE?

Reply:
![Screen Shot 2021-03-25 at 5 23 13 PM](https://user-images.githubusercontent.com/18150417/112559714-cb437400-8d8e-11eb-81db-5a91e78233fd.png)
This is what I see under OE, is this correct?

Reply:
You have to expand the server node. You should NOT be seeing the HDFS node for your servers

![image](https://user-images.githubusercontent.com/28519865/112560359-46595a00-8d90-11eb-85be-e6811f76ba18.png)

Reply:
yep, there is no HDFS node.

Reply:
![HDFS](https://user-images.githubusercontent.com/18150417/112566817-eae19900-8d9c-11eb-94bc-8804f11ca21e.PNG)